exporter/stackdriverexporter/factory.go

```diff
@@ -68,8 +68,8 @@ func createTraceExporter(
 // createMetricsExporter creates a metrics exporter based on this config.
 func createMetricsExporter(
 	_ context.Context,
-	_ component.ExporterCreateParams,
+	params component.ExporterCreateParams,
 	cfg configmodels.Exporter) (component.MetricsExporter, error) {
 	eCfg := cfg.(*Config)
-	return newStackdriverMetricsExporter(eCfg)
+	return newStackdriverMetricsExporter(eCfg, params.ApplicationStartInfo.Version)
```

Comment:
What about trace exporter? Do we need to add support for custom user agent to `opentelemetry-operations-go` first?

Reply:
I've documented that this only applies to metrics for now. We'll need to update the operations exporter before applying this to traces. Tracked here: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/82